about.css

```diff
@@ -0,0 +1,1149 @@
+}
+
+/* feature */
+.feature-speaker {
+  margin: 2rem;
+  display: grid;
+  margin-top: 5rem;
+}
+
+.single-feature-speker {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 1rem;
+  grid-template-columns: 1fr 1fr;
+  columns: 3rem;
+  width: 327px;
+  height: 200px;
+  margin-top: 2rem;
+}
+
+.image {
+  position: relative;
+}
+
+.speaker-description {
+  margin-left: 2rem;
+}
+
+.speaker-name {
+  padding-bottom: 0.5rem;
+  font-size: 0.9rem;
+}
+
+.speaker-details {
+  color: #ec5242;
+  font-size: 0.8rem;
+  font-style: italic;
+}
+
+.border-line1 {
+  display: none;
+}
+
+.border-line {
+  border-bottom: 1px solid;
+  width: 40px;
+  height: 1px;
+  background-color: #969595;
+  margin: 4px;
+}
+
+.speaker-background {
+  color: #272a31;
+  font-size: 0.8rem;
+}
+
+.bg-img {
+  position: absolute;
+  width: 70px;
+  z-index: -1;
+  top: -12%;
+  left: -20%;
+}
+
+.avatar {
+  display: block;
+  width: 130px;
+  height: 160px;
+}
+
+.FeauredSpeakers {
+  display: flex;
+  justify-content: center;
+  margin-top: 6rem;
+}
+
+hr {
+  width: 100px;
+  color: #e53d3d;
+  height: 7px;
+  margin-left: 150px;
+}
+
+.hidde {
+  display: none;
+}
+
+/* feature-speaker end  */
+
+/* partner section */
+
+#partner {
+  background-color: #515356;
+  padding: 2rem;
+}
+
+.partner-heading {
+  text-align: center;
+  color: wheat;
+}
+
+.partners-logos-container {
+  margin: 1rem 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  gap: 2rem;
+  margin-top: 2rem;
+}
+
+.partners-logo {
+  width: 100px;
+}
+
+/* .partner section end  */
+
+/* footer section */
+#footer {
+  padding: 2rem;
+  display: flex;
+  justify-content: space-between;
+  height: 100px;
+}
+
+.left-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.logo-img {
+  width: 110px;
+  height: 45px;
+  margin-right: 0.5rem;
+  margin-top: 2rem;
+}
+
+.footer-heading,
+.footer-description {
+  margin-left: 1rem;
+}
+
+.footer-heading {
+  font-size: 0.8rem;
+  color: #2b2b2b;
+}
+
+.footer-description {
+  font-size: 0.7rem;
+  color: #090909;
+}
+
+/* about page */
+
+.about-heading {
+  text-align: center;
+  padding: 2rem 0 0 1rem;
+  width: 80%;
+  color: #ec5242;
+}
+
+.about-description {
+  margin: 2.5rem auto 0;
+  background-color: rgb(212, 209, 209);
+  padding: 6%;
+  border: 1px solid var(--border-lines-color);
+}
+
+.contact-text {
+  background-color: rgb(212, 209, 209);
+  text-align: center;
+  font-size: 1.1rem;
+  padding: 1.5rem 2rem;
+  color: var(--main-text-color);
+}
+
+.email-contact {
+  background-color: rgb(212, 209, 209);
+  font-size: 1rem;
+  text-decoration: underline;
+  text-align: center;
+}
+
+#logo-section {
+  background-color: #fff;
+  padding: 2rem 0;
+}
+
+.logo-heading,
+.logo-description {
+  text-align: center;
+  color: var(--main-text-color);
+  padding: 0.5rem 1rem;
+}
+
+.logo-heading {
+  margin: 0 auto;
+  max-width: 80%;
+}
```

Comment:
- [ ] Please kindly make sure there is a divider after the title 😉

Your design
<img width="390" alt="Screen Shot 2023-01-14 at 04 26 58" src="https://user-images.githubusercontent.com/74501523/212444339-0f1c48a0-5ca0-4cb1-bde3-9180ee28c9d1.png">

Original design
<img width="544" alt="Screen Shot 2023-01-14 at 04 29 53" src="https://user-images.githubusercontent.com/74501523/212444386-5a3a7151-6e8b-4b95-a8ec-3e1f3c901989.png">